api/contract/cli.yml

```
@@ -81,6 +81,14 @@ paths:
     $ref: "./openapi/src/common/paths/dbrps.yml"
   "/dbrps/{dbrpID}":
     $ref: "./openapi/src/common/paths/dbrps_dbrpID.yml"
+  /authorizations:
+    servers:
+      - url: "/legacy"
+    $ref: "./openapi/src/common/paths/authorizations.yml"
+  /authorizations/{authID}:
+    servers:
+      - url: "/legacy"
+    $ref: "./openapi/src/common/paths/authorizations_authID.yml"
```

This still won't resolve to the right thing, it'll become `/legacy/authorizations/<stuff>`. This should do it:
```suggestion
  /legacy/authorizations:
    servers:
      - url: "/private"
    $ref: "./openapi/src/common/paths/authorizations.yml"
  /legacy/authorizations/{authID}:
    servers:
      - url: "/private"
    $ref: "./openapi/src/common/paths/authorizations_authID.yml"
```